controllers/snapshot/snapshot_adapter.go

```diff
@@ -501,10 +501,18 @@ func (a *Adapter) createSnapshotEnvironmentBindingForSnapshot(application *appli
 		environment.Name,
 		snapshot, *components)
 
-	// set environment as owner of snapshotEnvironmentBinding on controlled
-	err := ctrl.SetControllerReference(environment, snapshotEnvironmentBinding, a.client.Scheme())
-	if err != nil {
-		return nil, err
+	// If ephemeral, set environment as owner of snapshotEnvironmentBinding on controlled,
```

Comment:
This comment should explain why instead of rephrasing what's written in the code

Reply:
Ack, updated.